docs/Makefile

```diff
@@ -3,15 +3,10 @@
 
 # You can set these variables from the command line.
 SPHINXOPTS    =
-SPHINXBUILD   = sphinx-build
+SPHINXBUILD   = python -m sphinx
 PAPER         =
 BUILDDIR      = _build
 
-# When both are available, use Sphinx 2.x for autodoc compatibility.
-ifeq ($(shell which sphinx-build2 >/dev/null 2>&1 ; echo $$?),0)
-	SPHINXBUILD = sphinx-build2
-endif
```

Comment:
Hmm... on my Arch system where `python` is 3.x, I'm afraid this will crash. We may still need to detect which Python executable to use.